addons/mail/static/src/xml/followers.xml

```diff
@@ -36,12 +36,12 @@
     -->
     <t t-name="mail.Followers.partner">
         <div role="menuitem" class="o_partner">
-            <a class="dropdown-item o_mail_redirect"
+            <a t-attf-class="dropdown-item o_mail_redirect {{ record.active === false ? 'o_inactive_partner': '' }}"
                href="#"
                t-att-title="record.name"
                t-att-data-oe-model="record.res_model"
                t-att-data-oe-id="record.res_id"><t t-esc="record.name"/></a>
-            <img t-att-src="record.avatar_url" alt="Avatar"/>
+            <img t-att-src="record.avatar_url" alt="Avatar" t-attf-class="{{ record.active === false ? 'o_inactive_partner_img': '' }}"/>
```

Comment:
I wonder if it wouldn't be cleaner to add a class on div.o_partner and adapt the css accordingly since you are using this condition twice. in the same dom element

Reply:
Done!